### PR TITLE
[IMP] pylint: Enable check of cyclomatic complexity

### PR DESCRIPTION
**Pull request by moylop260**
We have next [guideline](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms):
`If a function is too long or too indented due to loops, this is a sign that it needs to be refactored into smaller functions`

The unit of measure of [cyclomatic complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity) can to help us to check it

Before of enable this check I need to know if is a good idea for you.

### CODE REVIEW

**Comment by moylop260** [comment]
FYI this check is enabled in informative checks of `travis_run_pylint_pr.cf` file without affect travis build result.

**Comment by moylop260** [comment]
ping

File: travis/cfg/travis_run_pylint_pr.cfg [hunk 1]
```diff
@@ -42,6 +43,7 @@ enable=api-one-multi-together,
     no-utf8-coding-comment,
     openerp-exception-warning,
     rst-syntax-error,
+    too-complex,
```
Comment (dreispt) [inline travis/cfg/travis_run_pylint_pr.cfg]: A too complex function will break the build, won't it?

Comment (moylop260) [inline travis/cfg/travis_run_pylint_pr.cfg]: Currently all pr.cfg messages are really beta mssages:
https://github.com/OCA/maintainer-quality-tools/issues/254
(Without affect travis status build).

Then too-complex checks don't affect travis build status too.

If we use the proposal process, I will create a new pr to add beta.cfg, where we will add too-complex.

Comment (dreispt) [inline travis/cfg/travis_run_pylint_pr.cfg]: ok

**Comment by dreispt** [comment]
:+1:

**Comment by moylop260** [comment]
Thanks @dreispt

**Comment by moylop260** [comment]
rebased

**Comment by oscarolar** [comment]
Excelent! :+1:

**Comment by moylop260** [comment]
@pedrobaeza

**Comment by moylop260** [comment]
ping

**Comment by pedrobaeza** [comment]
:+1:

**Comment by moylop260** [comment]
thaks pedro

**Pull request merged**